multidim-interop/src/compose-runner.ts

```diff
@@ -43,14 +44,11 @@ export async function run(namespace: string, compose: ComposeSpecification, opts
     }
 
     try {
-        const timeoutSecs = 3 * 60
-        let timeoutId
-        const { stdout, stderr } =
-            (await Promise.race([
-                exec(`docker compose -f ${path.join(dir, "compose.yaml")} up ${upFlags.join(" ")}`),
-                // Timeout - uses any type because this will only reject the promise.
-                new Promise<any>((resolve, reject) => { timeoutId = setTimeout(() => reject("Timeout"), 1000 * timeoutSecs) })
-            ]))
+        const timeoutId = setTimeout(() => controller.abort(), 1000 * timeoutSecs)
+
+        const controller = new AbortController();
```

Comment:
Isn't `controller` undefined in the closure? Even if this works, can we declare `controller` first please?

Reply:
It's tricky, but L49 will always happen before the closure. I'll update this anyways.